packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -303,8 +303,8 @@ class _BottomNavigationTile extends StatelessWidget {
 }
 
 class _BottomNavigationBarState extends State<BottomNavigationBar> with TickerProviderStateMixin {
-  List<AnimationController> _controllers;
-  List<CurvedAnimation> _animations;
+  List<AnimationController> _controllers = <AnimationController>[];
+  List<CurvedAnimation> _animations = <CurvedAnimation>[];
```

Comment:
Looks like the initial value for `_animations` is never used?

Reply:
Good point, I've removed the initial value.